gateway/src/apicast/policy/rate_limit/rate_limit.lua

```diff
@@ -160,6 +161,7 @@ function _M.new(config)
   self.redis_url = config.redis_url
   self.error_settings = init_error_settings(
     config.limits_exceeded_error, config.configuration_error)
+  self.timestamp = ngx.time()
```

Comment:
There is no need to store time in `self.timestamp`. Local variable will be ok too.

Reply:
Fixed.